solidity/test/proof/Sumcheck.t.pre.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: UNLICENSED
+// This is licensed under the Cryptographic Open Software License 1.0
```

Comment:
@JayWhite2357 Hmm I wonder whether we should add our license here: https://spdx.org/licenses/